aem-authoring-toolkit-assets/src/main/content/jcr_root/apps/authoring-toolkit/depends-on/README.md

```diff
@@ -141,7 +143,7 @@ Single reference should reference existing field and will not be reattached on d
 Note: multiple reference triggers query update on any group update: changing some of group fields value, adding or removing referenced field. 
 So usage of multiple reference can slow down queries performance.
 
-Reference can not be named as 'this', that name is reserved and always reach current element value.
+Reference cannot be named 'this', this name is reserved and always accesses current element value.
```

Comment:
Reference cannot be named 'this', because 'this' is reserved to refer to the value of the current element

Reply:
Thanks, looks much more readable!